docs/releases/2.2.4.rst

```diff
@@ -0,0 +1,66 @@
+Translate Toolkit 2.2.4
+***********************
+
+*Released on 31 July 2017*
+
+This release contains improvements and bug fixes.
+
+
+Major changes
+=============
+
+- Full support for `.xliff` extension.
+- Added support for nested and WebExtension JSON dialects.
+- `po2txt` no longer converts non-translatable strings.
+- Improvement for `puncspace` check.
+
+
+Detailed changes
+================
+
+Formats and Converters
+----------------------
+
+- All formats
+
+  - Added support for `.xliff` extension in all converters that support `.xlf`
+    extension.
+
+- JSON
+
+  - Added support for nested JSON.
+  - Added support for WebExtension JSON dialect.
+
+- txt
+
+  - `po2txt` skips obsolete and non-translatable strings.
```

Comment:
No, my description is correct according to the commit messages, the code changes and the tests you landed. In other words, `po2txt` now correctly skips header and obsolete units from PO files when converting back to txt format.